x-pack/docs/en/security/authentication/service-accounts.asciidoc

```diff
@@ -0,0 +1,113 @@
+[role="xpack"]
+[[service-accounts]]
+=== Service accounts
+
+beta::[]
+
+The {stack-security-features} provide _service accounts_ for better integration
+with external services. Service accounts are an evolution of the {es}
+<<built-in-users,built-in users>>, with notable differences. While built-in users work almost
+identically to users that you create, service accounts have a fixed set of
+privileges and cannot be authenticated until you create a service account token
+for them. Additionally, service accounts are predefined in code, and are always
+enabled.
+
+With service accounts, services (such as {fleet}) can create a service account
+token, authenticate with that token, and manage their own API keys. Service
+accounts provide flexibility over built-in users because they:
+
+* Do not rely on the <<native-realm,internal `native` realm>>, and aren't
+always required to rely on the `.security` index
+* Use a role descriptor named after the service account principal instead of traditional roles
+* Support multiple credentials through service account tokens
+
+Service accounts are not included in the response of the
+<<security-api-get-user,get users API>>. To retrieve a service account, use the
+<<security-api-get-service-accounts,get service accounts API>>.
+
+[[service-accounts-explanation]]
+==== How service accounts work
+Service accounts have a
+<<security-api-get-service-accounts-path-params,unique principal>> that takes
+the format of `<namespace>/<service>`, where the `namespace` is a top-level
+grouping of service accounts, and `service` is the name of the service.
+
+Currently, only one service account is available:
+
+`elastic/fleet-server`:: The service account used by the {fleet} server to
+communicate with {es}.
+
+// tag::service-accounts-usage[]
+IMPORTANT: The predefined service accounts are specifically for integrating with
+external services and are not intended for general use.
+// end::service-accounts-usage[]
+
+// tag::service-accounts-tls[]
+In <<dev-vs-prod-mode,production mode>>, service accounts require TLS on the
+HTTP interface. A runtime check prevents you from invoking any related APIs or
+authenticating with a service account token unless TLS is enabled on the HTTP
+interface. See <<encrypt-http-communication,encrypt HTTP client communications for {es}>>.
+// end::service-accounts-tls[]
+
+NOTE: Service accounts currently do not support basic authentication.
```

Comment:
I feel that having this note here may be a bit confusing, as service account tokens are not explained yet.
Would it make sense to move down in the page after "Authenticate with service account tokens"?

Reply:
@bytebilly, should this note indicate that:
>Service account **tokens** currently do not support basic authentication

Or is it the service accounts themselves that do not support basic authentication?

Reply:
It's about service accounts, I read it as "service accounts don't have a password".
My point is that we should say what we support first (tokens), and then, for people wondering, what we don't (basic auth).I think that otherwise it may be unclear for people that are not fully familiar with the technical differences between the two.

Reply:
@bytebilly, good point. I moved that sentence to the section about authenticating with service account tokens.

Reply:
LGTM 👍